_CONTRIBUTING.md

```diff
@@ -1,4 +1,34 @@
-# Contributing
+# Contributing/Git Workflow
+
+## Team Workflow
+1. git checkout development
+2. git pull --rebase upstream development
```

Comment:
Since we're working out of the same repo, I think this should be changed to: git pull --rebase origin development